chart/kubeapps/templates/tests/test-chartsvc.yaml

```diff
@@ -16,5 +16,5 @@ spec:
     command:
     - sh
     - -c
-    - curl $CHARTSVC_HOST:$CHARTSVC_PORT/v1/charts | grep wordpress
+    - curl -o /tmp/output $CHARTSVC_HOST:$CHARTSVC_PORT/v1/charts && cat /tmp/output && cat /tmp/output | grep wordpress
```

Comment:
Why are you `cat`ing the whole /tmp/output then also redoing with the pipe? (here and everywhere).
That just means we see some of the output twice, no?

Reply:
If we just do the `grep` pipe and the output doesn't contain what we expect it to contain (like for example an error), the actual content would be filtered out. That's why I am showing the output and then do the "grep" as the test.